### PR TITLE
Add a new date-only SAM floating histogram visualization plugin

### PR DESCRIPTION
**Pull request by bobular**
Works toward https://github.com/VEuPathDB/web-monorepo/issues/1026

Sorry, I'm not to clear about what happens in AppsService.java.  There's some autogenerated model magic that I can't follow in the source code as-is. (@d-callan helped me out with this - hopefully all sorted now)

### CODE REVIEW

File: api.raml [hunk 1]
```diff
@@ -834,6 +834,18 @@ securitySchemes:
               application/json:
                 type: lib.FloatingLineplotPostResponse
 
+    /timeline:
```
Comment (d-callan) [inline api.raml:837]: this seems easy to confuse w timeseries

File: src/main/java/org/veupathdb/service/eda/data/metadata/AppsMetadata.java [hunk 2]
```diff
@@ -75,6 +76,7 @@ public class AppsMetadata {
           List.of(VECTORBASE_PROJECT),
           viz("lineplot", new FloatingLineplotPlugin()),
           viz("timeseries", new FloatingTimeSeriesPlugin())),
+          viz("timeline", new FloatingDateHistogramPlugin()),
```
Comment (d-callan) [inline src/main/java/org/veupathdb/service/eda/data/metadata/AppsMetadata.java:79]: seems like this should belong to distributions ?

File: api.raml (hunk 1 above)
Comment (d-callan) [reply api.raml:837]: also seems it should move to distributions here as well

**Comment by bobular** [comment]
It compiles!

```
BUILD SUCCESSFUL in 17s
4 actionable tasks: 4 executed
```

**Review by d-callan — APPROVED** [review]

**Pull request merged**